### DISCUSSION TITLE
[Feature] Immich as the only galary app (allow deleting photos locally and not from the server)

### DISCUSSION THREAD

**Discussion by anisjonischkeit** [immich-app/immich#9192] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would love to use immich as the only gallary app on mobile. I want to be able to backup everything to my server, but then only keep a subset of images locally to free up space. It would be great to by default only delete photos locally (keep them on the server) and add an additional confirm step if you want to delete the photos permanently.

In my case, I would even want to go as far as to disallow deleting server files via mobile entirely (or ask for a harder confirmation like a pin or an input where you have to type "delete permanently" or something)

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1605
